pydal/_load.py

```diff
@@ -5,10 +5,12 @@
     from collections import OrderedDict
 except:
     from .contrib.ordereddict import OrderedDict
+    
 try:
+# try external module
+    import simplejson as json
```

Comment:
as far as I remember, the external module (if present) is faster

 Paolo

2015-04-27 0:04 GMT+02:00 Giovanni Barillari <notifications@github.com>:

> In pydal/_load.py
> <https://github.com/web2py/pydal/pull/147#discussion_r29114486>:
>
> >  try:
> > +# try external module
> > +    import simplejson as json
>
> why keep trying to load simplejson if we're dropping py25?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/web2py/pydal/pull/147/files#r29114486>.
>